### ISSUE TITLE
MGT tank trinket procs on 99.9% of druid hp

### ISSUE THREAD

**Issue by NeonicSupersonic** [ValorenWoW/sunstrider-core#4523]
MGT tank trinket effect "gain x dodge rating when below 35% hp" procs on druid in bear form when he receives any amount of damage which brings him below 100% hp(even if it is absolutely minimal amount of damage). Must have something to do with using normal druid form hp which is set to 1 or 0 hp when druid is in the bear form for trinket trigger check.

**Comment by kelno** [comment]
Thanks!
Good guess but I can't reproduce this, I have a 10k hp druid and the trinket trigger around 3k5 correctly.
Any more details on what could be different for you here? What's the character name?

**Comment by NeonicSupersonic** [comment]
Very strange that it is not reproducible, was happening through all of the bt on our drood tank including fr tanking on illidan and whatsnot so I have assumed that it was a global bug. Now I have no idea,
https://endless.gg/armory/character/notor

**Comment by kelno** [comment]
Okay we found how to reproduce it, it has to do with the trinket checking only after the damage was already done if new damage would take the owner under 35%. So basically considering the hit was twice as strong. 
Fixed with next server update.
